src/app/services/form.service.ts

```diff
@@ -47,4 +48,12 @@ export class fromService {
         : this.currentdate.getDate();
     return year + "-" + month + "-" + day;
   }
+
+  setList(list?) {
+    if (list) {
+      localStorage.setItem("list", JSON.stringify(list));
```

Comment:
can you explain this function. why is this "list" and this.todos"

doesn't make this thing.

Reply:
i don't think if condition should be applied on this function, rather should be applied where the function is called

Reply:
set list is called form most of the places without argument and in  one place  set list call with argument